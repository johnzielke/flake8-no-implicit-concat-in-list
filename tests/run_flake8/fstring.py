v = "vvv"
a = [f"aaa{v}"
     "bbb"]
print(a)
