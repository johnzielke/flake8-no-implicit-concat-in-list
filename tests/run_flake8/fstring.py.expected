./fstring.py:2:15: ICL121 Implicitly concatenated str literals in list literal over multiple lines.
